libs/application/templates/financial-aid/src/components/Logo/Logo.tsx

```
@@ -0,0 +1,29 @@
+type Props = {
+  application: Application
+}
+export const Logo = ({ application }: Props) => {
+  const [logo, setLogo] = useState<string>()
+  const municipality = application.externalData.municipality
+    ?.data as Municipality
+
+  useEffect(() => {
+    const getLogo = async () => {
+      const municipalityId =
+        municipality && municipality?.municipalityId
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)